.circleci/config.yml

```diff
@@ -296,27 +302,17 @@ jobs:
     executor: docker
     resource_class: large
     steps:
+      - full-checkout
       - restore-sccache-cache
       - test-setup
       - run: rustup override set beta
       - run-tests
       - save-sccache-cache
-  Rust benchmarks:
-    executor: docker
-    resource_class: large
-    steps:
-      - restore-sccache-cache
-      - bench-all
-      - save-sccache-cache
   iOS build and test:
     executor: macos
     steps:
-      # We do not use the ssccache cache helper commands as
-      # the macOS cache is in a different folder.
-      - restore_cache:
-          name: Restore sccache cache
-          key: sccache-cache-macos-{{ arch }}
       - full-checkout
+      - restore-sccache-cache
```

Comment:
the `{{ arch }}` already differentiates between the caches for different machines so this can use the same `restore-sccache-cache` command the other docker runner use, so I changed it